### ISSUE TITLE
LibreOffice 7.0 Calc crash creates zombie stacked window

### ISSUE THREAD

**Issue by tcc-sejohnson** [pop-os/shell#920]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="20.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.10"
VERSION_ID="20.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=groovy
UBUNTU_CODENAME=groovy
LOGO=distributor-logo-pop-os
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
N/A, I think? Whatever version of LibreOffice Calc (7.0?) is included with Pop.

**Issue/Bug Description:**
I had a really big CSV I was trying to import. Like... ~1m rows. Trying to open it caused LibreOffice to hang, which also crashed all GUI but left my mouse functional. I used `ctrl`+`alt`+`f5` to open a virtual terminal and kill Libre. When I logged back into the GUI, I got the "A process is not responding, wait/kill" dialogue, but it was completely frozen. I used Super to switch to a different application, which worked. However, I now have a zombie stacked window that cannot be Super+Arrowkeyed over or closed. 

**Steps to reproduce (if you know):**
I can reproduce this issue about 30% of the time by repeating the above steps. The other 70% of the time, the GUI crashes irrecoverably (i.e. I can't even go into the virtual terminal and have to "raise the elephant"). 

**Expected behavior:**
A crashed program should, when terminated, not have a tab in the stack menu anymore.

**Other Notes:**
Here's a screenshot of the zombie tab. It cannot be selected, arrowed over, or gotten rid of in any way I know of, short of a reboot.

![image](https://user-images.githubusercontent.com/76245373/113318343-9e7ee780-92cd-11eb-9f72-c85a4b9f6d8d.png)

**Comment by tcc-sejohnson** [comment]
@bflanagin Yeah, I didn't expect you to be able to do anything about Libre -- I'm more concerned with the zombie tab. I use stacking a lot on my laptop, and it's one of only a couple bugs that has been annoying me. Happy to reproduce and provide whatever logs you need as well.

**Comment by mmstick** [comment]
Will need to create a minimalistic GTK application that segfaults at will to debug this

**Comment by tcc-sejohnson** [comment]
FYI, I ran into this issue again today. When searching about it, I found my issue again. 🙂
